.github/workflows/publish-on-pypi.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish release to PyPI
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  build-and-upload:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the release commit
+      uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
```

Comment:
I'd suggest using a more recent version, so that we don't have to update this workflow when Python 3.8 reaches EOL next year.